.github/workflows/comment-npm-publish.yml

```diff
@@ -42,7 +42,7 @@ jobs:
 
       - uses: actions/setup-node@v3
         with:
-          node-version: 14
+          node-version-file: ".nvmrc"
```

Comment:
We might need to add guards, just like the dependency-tree workflow: https://github.com/Tradeshift/actions-workflow-npm/blob/master/.github/workflows/dependency-tree-update.yml#L49-L66
- Ignore the middle step there, but a check for `nvmrc` exists, and a fallback.

Reply:
I updated